README.md

```diff
@@ -18,28 +18,34 @@
     var express = require('express');
     var app = express();
 
-    // create the view engine with `react-engine`
-    var engine = require('react-engine').server.create({
-      reactRoutes: <string> /* pass in the path to react-router routes optionally */
-      performanceCollector: <function> /* optional function to collect perf stats */
-    });
+    app.use(function(req, res) {
```

Comment:
@sherodtaylor thanks for the PR. why do we need to run this code piece like a middleware?